service/storage_service.cc

```diff
@@ -6127,7 +6131,6 @@ void storage_service::init_messaging_service(bool raft_topology_change_enabled)
                     auto read_apply_mutex_holder = co_await ss._group0->client().hold_read_apply_mutex();
                     topology_requests_mutations = co_await ss.get_system_mutations(db::system_keyspace::NAME, db::system_keyspace::TOPOLOGY_REQUESTS);
                 }
-                }
```

Comment:
and now you remove it -- let's not introduce it in the first place (the commit where it sneaked in)

Generally, each commit should in theory compile and even pass all tests (although we don't enforce the latter that much)

Reply:
yes, squashed to wrong commit that's why it happened